schedview/compute/visits.py

```diff
@@ -0,0 +1,227 @@
+import datetime
+
+import numpy as np
+from astropy.time import Time
+from rubin_scheduler.site_models import SeeingModel
+from rubin_sim import maf
+
+import schedview.compute
+
+
+def add_day_obs(visits):
+    """Add day_obs columns to a visits DataFrame.
+
+    Parameter
+    ---------
+    `visits` : `pandas.DataFrame`
+        The DataFrame of visits to which to add day_obs columns
+
+    Returns
+    -------
+    `visits` : `pandas.DataFrame`
+        The modified DataFRame with additonal columns: day_obs_date,
+        day_obs_mjd, and day_obs_iso8601.
+    """
+    day_obs_mjd = np.floor(visits["observationStartMJD"] - 0.5).astype("int")
+    day_obs_datetime = Time(day_obs_mjd, format="mjd").datetime
+    day_obs_date = [datetime.date(t.year, t.month, t.day) for t in day_obs_datetime]
+    day_obs_iso8601 = tuple(str(d) for d in day_obs_date)
+    visits.insert(1, "day_obs_mjd", day_obs_mjd)
+    visits.insert(2, "day_obs_date", day_obs_date)
+    visits.insert(3, "day_obs_iso8601", day_obs_iso8601)
+    return visits
```

Comment:
We might want to go ahead and also incorporate the summit tools as long they don't require the stack to install. And use Merlin's computation of DAYOBS there. Then we would always stay in sync with the same definition of DAYOBS, in case it changes (due to taking flats or other calibration data in the morning, for example).